docs/resources/cluster.md

```diff
@@ -1614,8 +1614,8 @@ ClusterSecrets defines cluster secrets.
 The following arguments are supported:
 
 - `docker_config` - (Optional) - (Sensitive) - String - DockerConfig holds a valid docker config.<br />After creating a dockerconfig secret, a /root/.docker/config.json file will be added to newly created nodes.<br />This file will be used by Kubernetes to authenticate to container registries and will also work when using containerd as container runtime.
-- `cluster_ca_cert` - (Optional) - (Sensitive) - String
-- `cluster_ca_key` - (Optional) - (Sensitive) - String
+- `cluster_ca_cert` - (Optional) - (Sensitive) - (Computed) - String
+- `cluster_ca_key` - (Optional) - (Sensitive) - (Computed) - String
```

Comment:
I assume it's still possible to pass a CA cert in as previously even if it's marked as computed?

Reply:
It should be possible to pass the certificate, it should only tell terraform that if the attribute is not provided by the user it will be computed server side.

I'm cutting a prerelease to test it before making a patch release though.

Reply:
Just built it locally and it solved my problem at least. 🎉 🙏🏻

Reply:
Thanks for trying it out, let me know if you have issues when passing a CA cert.